src/03-deals/deals.apib

```diff
@@ -74,6 +74,7 @@ Get a list of deals.
                 + department (object)
                     + type: `department`
                     + id: `33121d39-44e2-426a-92ef-62178edeec8a` (string)
+                + weighted_value (Money)
```

Comment:
PP: this property is a calculated one based on values in `estimated_value` and `estimated_probability`, so it feels more natural to place it below those.